schema/crdb/34.0.0/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE UNIQUE INDEX IF NOT EXISTS network_interface_parent_id_slot_key ON omicron.public.network_interface (
```

Comment:
This is correct. We effectively had a uniqueness guarantee from the `NextItem` subquery, but this is the right way to enforce that when the user can also provide one explicitly.